Microcontroller_L432KC/README.md

```diff
@@ -0,0 +1,18 @@
+# ECE3SAT_PhaseB_EDT
+
+This code has been wrote for STM32 Nucleo L432KC.
+
+## Prerequisites
+
+You need to have an installed and configured [MbedCLI](https://os.mbed.com/docs/v5.7/tools/arm-mbed-cli.html).
+
+
+## Installation
+
+Use `mbed new .` in this directory to create a Mbed project with all configurations and dependencies in this repository.
+
+You can now use mbed command to compile these files to get the binary to flash your microcontroller

```

Comment:
"compile these files into a binary which can be flashed on your microcontroller" would be easier to comprehend imho.

Reply:
Also: is this command both compiling AND flashing or is it just compiling and you later have to flash it manually ? 
(Make it clear in the guiding text)